portal/src/main/webapp/js/src/study-view/data/StudyViewProxy.js

```diff
@@ -257,7 +257,7 @@ var StudyViewProxy = (function() {
                 }else {
                     var cnaLength = obtainDataObject.arr.length;
                     for(var i = 0; i < cnaLength; i++) {
-                        if(obtainDataObject.arr[i]['MUTATION_COUNT']) {
+                        if(!_.isUndefined(obtainDataObject.arr[i]['MUTATION_COUNT'])) {
```

Comment:
In line 258, should variable be called cnaLength or something like mutationLength? 

I would also advocate for keeping the same style in referring to attributes, so either ['MUTATION_COUNT'] OR .MUTATION_COUNT. Now you use both mixed up in this and next line.